js/app.js

```diff
@@ -23,28 +23,37 @@ function getRez() {
 
                 const wrp = document.createElement("div")
                 wrp.className = "wrp"
+                let ls = r.languages.map(l => {
```

Comment:
this function takes the languages array from each country and returns each language's name.